File: app/src/main/java/com/microlabs/trallet/view/AddCurrencyActivityView.java
package com.microlabs.trallet.view;

/**
 * Created by Hansen on 5/11/2017.
 * <p>
 * Add Currency Activity View
 */

public interface AddCurrencyActivityView {
    void showError();

    void done();

    void showDuplicateError();

    void validateDeleteCurrency();

    void showFailDeleteCurrency();
}

File: app/src/main/java/com/microlabs/trallet/view/AddCurrencyActivityView.kt
package com.microlabs.trallet.view

/**
 * Created by Hansen on 5/11/2017.
 *
 *
 * Add RCurrency Activity View
 */

interface AddCurrencyActivityView {
    fun showError()

    fun done()

    fun showDuplicateError()

    fun validateDeleteCurrency()

    fun showFailDeleteCurrency()
}
